pkg/api/v1/schema.go

```diff
@@ -170,6 +170,8 @@ type Devices struct {
 	Watchdog *Watchdog `json:"watchdog,omitempty"`
 	// Interfaces describe network interfaces which are added to the vm.
 	Interfaces []Interface `json:"interfaces,omitempty"`
+	// Whether to attach a pod network interface. Defaults to true.
+	AutoAttachPodInterface *bool `json:"autoAttachPodInterface,omitempty"`
```

Comment:
Not a huge thing  but I see that in k8s something equivalent is called `automountServiceAccountToken` and not `autoMountServiceAccountToken`. Do we want to use `autoattach` instead of `autoAttach`?

Reply:
Done.